coremltools/converters/mil/input_types.py

```diff
@@ -269,6 +269,11 @@ def __init__(self, lower_bound=1, upper_bound=-1, default=None,
                 )
             self.default = default
 
+    def __repr__(self):
+        return f'RangeDim({self.lower_bound}, {self.upper_bound}, {self.default}, "{self.symbol}")'
+
```

Comment:
Nice. 

Would it more readable if we also include parameter names in the string for user to know what each number refers to? something like `"RangeDim(lower_bound={self.lower_bound} ... "`

Reply:
Hey we should avoid f-string till release 4.0 moves out of beta.

Reply:
Agreed. So maybe should update this to:

```python
def __repr__(self):
    return "RangeDim(lower_bound={}, upper_bound={}, default={}, symbol={})".format(
            self.lower_bound, self.upper_bound, self.default, self.symbol)
```

@guy4261 Could you help to make this change? Thanks!

Reply:
Added double quotes (a-la `black`) to the symbol, because that's how you'd construct this object.

Also changed this to be the `__str__` and the `__repr__` to return the `__str__` (like the `TensorType` class just above it).

```python
class RangeDim(object):
    ...
    def __repr__(self):
        return self.__str__()

    def __str__(self):
        return 'RangeDim(lower_bound={}, upper_bound={}, default={}, symbol="{}")'.format(
            self.lower_bound, self.upper_bound, self.default, self.symbol)

>>> RangeDim(1,2)
RangeDim(lower_bound=1, upper_bound=2, default=1, symbol="is2")
```